json-rules-core/src/main/java/io/appform/jsonrules/utils/JsonUtils.java

```diff
@@ -0,0 +1,32 @@
+package io.appform.jsonrules.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.TextNode;
+import io.appform.jsonrules.config.JacksonConfiguration;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class JsonUtils {
+
+    public static Set<Object> convertToJsonNode(Set<Object> values) {
+        if (values.stream().allMatch(value -> value instanceof JsonNode)) {
+            return values;
+        }
+        ObjectMapper mapper = JacksonConfiguration.getInstance().getObjectMapper();
+        Set<Object> valuesAsJsonNodes = new LinkedHashSet<>();
+        values.forEach(value -> valuesAsJsonNodes.add(mapper.valueToTree(value)));
+        return valuesAsJsonNodes;
+    }
+
+    public static String convertToString(Object obj) {
+        if (obj == null) {
+            return "null";
```

Comment:
Sending `""` is better here

Reply:
I thought since this is only used for debugging wanted to set it to null as such since "" and null have different semantics. I can change it to "" if you think it is more appropriate.

Reply:
Leaving it "null" for now for the above reasons.